### DISCUSSION TITLE
[QUESTION] Any plans for mobile app with Flutter?

### DISCUSSION THREAD

**Discussion by rileyhawk1417** [AppFlowy-IO/AppFlowy#72] [Ideas]
Hi just to ask is there any plan for developing the mobile version of the app using the Flutter Framework?

**Comment by ghost** [comment]
[It's a long-term goal](https://trello.com/c/ksHlrelo)

**Comment by inson1** [comment]
@rileyhawk1417 now when there is mobile app, what about closing this question? :)
